backend/src/main/java/org/kaiteki/kaizen/api/README.md

```diff
@@ -0,0 +1,93 @@
+# Chatbot API
+
+A simple Flask API for interacting with a conversational AI model. This API allows you to perform various natural language processing tasks, such as chat-based responses, text summarization, keyword extraction, and paraphrasing.
```

Comment:
Rewrite readme for up to date information. its incorrect now, in response type and etc.

Better use swagger for documentation. In readme file only specify the guide on how to run project and important endpoints